sumy/nlp/tokenizers.py

```diff
@@ -100,13 +119,15 @@ class Tokenizer(object):
         'japanese': nltk.RegexpTokenizer('[^　！？。]*[！？。]'),
         'chinese': nltk.RegexpTokenizer('[^　！？。]*[！？。]'),
         'korean': KoreanSentencesTokenizer(),
+        'greek': GreekSentencesTokenizer(),
     }
 
     SPECIAL_WORD_TOKENIZERS = {
         'hebrew': HebrewWordTokenizer(),
         'japanese': JapaneseWordTokenizer(),
         'chinese': ChineseWordTokenizer(),
         'korean': KoreanWordTokenizer(),
+        'greek': GreekWordTokenizer(),
```

Comment:
Now I realized you could replace this with NLTK tokenizer.
```suggestion
        'greek': nltk.RegexpTokenizer(r'[ ,;;.\-!?:]+', gaps=True),
```